YoCode/UICheck.cs

```diff
@@ -1,21 +1,56 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Linq;
 
 namespace YoCode
 {
     public class UICheck
     {
-        public static bool UIContainsFeature(String userFileURL, String[] keyWords)
+        public UICheck(List<String> userFilePaths, String[] keyWords)
         {
-            var userFile = File.ReadAllText(userFileURL);
-            var comp = StringComparison.OrdinalIgnoreCase;
+            UIContainsFeature(userFilePaths, keyWords);
+        }
+
+        public UICheck(String userFilePath, String[] keyWords)
```

Comment:
In C#, it's more common to use the the keyword `string` rather than the class name `System.String` (just like it's more common to use `int` over `System.Int32`, etc.). They mean the same thing, but it looks a little odd a C# codebase to see `String` used for the types.